vars

```diff
@@ -0,0 +1 @@
+pipeline_steps/
```

Comment:
Thats a symlink from vars --> pipeline steps. This is required so that rpc-gating can be loaded a shared library in Jenkins. See https://jenkins.io/doc/book/pipeline/shared-libraries/#directory-structure for the structure.